### PR TITLE
Completed css-flexbox exercise.

### PR DESCRIPTION
**Pull request by MohamadAlsyouf**
image
image
image
image

### CODE REVIEW

File: css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - align-items</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck {
+        display: flex;
+        flex-basis: calc(100% / 3);
```
Comment (codingcodymiller) [inline css-flexbox/align-items.html:14]: `flex-basis` is a property that is applied to children of a flexbox, not the flexbox itself. This should be being applied to `.card-wrapper`

**Review by codingcodymiller — APPROVED** [review]

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Misclicked, sorry!

**Comment by MohamadAlsyouf** [comment]
image

**Comment by MohamadAlsyouf** [comment]
accidentally merged initial commit before adding the changes. I added them, commited, and pushed this exercise again, do I need to do anything since I merged it before these changes or am I cool?

**Pull request merged**